claim.md

```diff
@@ -52,10 +56,13 @@ Append the following code to `2-claim.js`:
 ```javascript
 /* RequestForAttestation needs 3 arguments: 
 * a claim
-* a legitimation (only useful for trust hierarchies, but for this tutorial we stick to a simple case)
 * a claimer
 */
-const requestForAttestation = new Kilt.RequestForAttestation(claim, [], claimer)
+const requestForAttestation = Kilt.RequestForAttestation.fromClaimAndIdentity(
```

Comment:
Suggestion: add `null` as a fourth argument, just for the sake of illustrating the API (see `fromClaimAndIdentity`, it takes 4 arguments)

Reply:
Updated to include Null